glide-core/redis-rs/redis/src/cluster_async/connections_container.rs

```diff
@@ -23,58 +24,66 @@ macro_rules! count_connections {
     }};
 }
 
-/// A struct that encapsulates a network connection along with its associated IP address.
+/// A struct that encapsulates a network connection along with its associated IP address and AZ.
 #[derive(Clone, Eq, PartialEq, Debug)]
-pub struct ConnectionWithIp<Connection> {
+pub struct ConnectionDetails<Connection> {
     /// The actual connection
     pub conn: Connection,
-    /// The IP associated with the connection
+    /// The IP and AZ associated with the connection
```

Comment:
The AZ is stored on a different parameter (the below it) - not this one

Reply:
you're right, a leftover from my attempt to make it tuple